components/hint-text/src/index.js

```diff
@@ -1,6 +1,6 @@
 // https://github.com/alphagov/govuk-frontend/tree/master/src/components/
 
-import glamorous from 'glamorous';
+import styled from 'react-emotion';
```

Comment:
Have reviewed as far as here... will continue once above feedback addressed

Reply:
Thanks so much for your time on this. Really appreciate it.